docs/en/Use-ShardingJDBC-as-storage-implementor.md

```diff
@@ -0,0 +1,18 @@
+# Supported Database Sharding Storage
+Beside the default Elasticsearch storage, it also support the database sharding storage, it allow the users to store data in multiple databases.
+Note: it only supported MYSQL database sharding, and due to the license restrictions, the users need to import MYSQL Driver manually.
+
+## Requirement
+- Manually import MySQL Driver package mysql-connector-java-5.1.36.jar to collector libs directory.
+- In config/application.yml, close the elasticsearch configuration and open the shardingjdbc configuration, multiple data source configurations should be separated by Half Comma.
```

Comment:
`In config/application.yml, close the ElasticSearch configuration and open the shardingjdbc configuration` -> `In config/application.yml, delete the ElasticSearch configuration and add the following shardingjdbc configuration`

Remove ~~multiple data source configurations should be separated by Half Comma.~~